config.yml

```diff
@@ -32,46 +34,50 @@ trading_options:
     - DOWN
     - UP
 
-  # Maximum number of trade 'slots' at any time (your USDT balance must be at least MAX_COINS * QUANTITY)
-  MAX_COINS: 7
+  # Maximum number of trade 'slots' at any time (your USDT balance must be at least TRADE_SLOTS * QUANTITY)
+  TRADE_SLOTS: 999
 
   # the amount of time in MINUTES to calculate the difference from the current price (minimum: 1).
-  TIME_DIFFERENCE: 2
+  TIME_DIFFERENCE: 3
 
   # Number of times to check for TP/SL during each TIME_DIFFERENCE (minimum: 1).
   # Don't spam the Binance API, you will be banned (max 1200 requests per minute per IP).
-  RECHECK_INTERVAL: 10
+  RECHECK_INTERVAL: 36
 
   # the difference in % between the first and second checks for the price.
-  CHANGE_IN_PRICE: 10
+  CHANGE_IN_PRICE: 2
 
   # define in % when to sell a coin that's not making a profit.
-  STOP_LOSS: 5
+  STOP_LOSS: 99
 
   # define in % when to take profit on a profitable coin.
-  TAKE_PROFIT: .8
+  TAKE_PROFIT: .7
 
   # Use custom tickers.txt list for filtering pairs.
   CUSTOM_LIST: True
-   
+
   # Name of custom tickers list
-  TICKERS_LIST: 'tickers.txt'
-   
+  TICKERS_LIST: 'tickers_ETH.txt'
```

Comment:
Please dont use ETH as base currency, set this all back to default, it will not conform with documentation

Reply:
have reset config values and have a separate config file now, thank you